### PR TITLE
Bug 991671 - Provide a simple way to run the python integration tests (a...

### PR DESCRIPTION
**Pull request by JohanLorenzo**
...ka UI Tests)

### CODE REVIEW

**Comment by try-server-hook** [comment]
[JohanLorenzo Johan Lorenzo (JohanLorenzo)](https://github.com/JohanLorenzo) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=bdf2ed8e6eb2)

File: bin/gaia-ui-tests [hunk 1]
```diff
@@ -3,6 +3,8 @@
 export GAIATEST_ACKNOWLEDGED_RISKS=true
 export GAIATEST_SKIP_WARNING=true
 
+DIR="$( cd "$( dirname "$SOURCE" )" && pwd )"
```
Comment (julienw) [inline bin/gaia-ui-tests]: are you sure it's `$SOURCE` and not `$0` ?

File: Makefile [hunk 2]
```diff
@@ -773,6 +773,10 @@ test-integration-test:
 		--reporter $(REPORTER) \
 		--buildapp $(BUILDAPP)
 
+.PHONY: test-gaia-ui-test
+test-gaia-ui-test:
+	./bin/gaia-ui-tests
```
Comment (julienw) [inline Makefile]: I actually wasn't aware of this script as it's not described in mdn at all.

I don't really care about a make target; having a proper script is good enough for me.

(the change to really-clean is welcomed though)

File: bin/gaia-ui-tests [hunk 3]
```diff
@@ -43,11 +45,27 @@ cd $root/../ && \
...
+
 # Run the tests.
-python $gaiatest --app=b2gdesktop \
-          --binary=$b2g \
-          --profile=$profile \
-          --type=b2g \
-          --timeout=10000 \
-          --restart \
-          $@
+if [ "$BUILDAPP" == "device" ]; then
```
Comment (julienw) [inline bin/gaia-ui-tests]: do you know the difference between BUILDAPP and MARIONETTE_RUNNER_HOST ?

**Comment by JohanLorenzo** [comment]
Mass closing PRs. Gaia is not a supported project anymore.

**Pull request closed without merging**